sw/device/silicon_creator/manuf/keys/sival/README.md

```diff
@@ -0,0 +1,17 @@
+# CA Endorsement Keys
+
+Certificate Authority endorsement keys are are used to endorse the following
+certificate chains during personalization:
+1. DICE attestation certificate chains, and
+2. SKU specific certificate chains.
+
+The real (private) keys used for the SIVAL SKU are stored on offline HSMs. The
+matching public keys and certificates are checked into the repository.
+
+To use the private keys to endorse the certificates in benchtop provisioning
+flow, one must set the `PKCS11_MODULE_PATH` envar to point to the PKCS#11 shared
+library for the hardware token they are using. For example, if the private keys
+are stored on a Nitrokey, you would set the envar as follows:
+```sh
+export PKCS11_MODULE_PATH=/opt/nitrokey/lib/libsc-hsm-pkcs11.so
```

Comment:
Note: you can use `opensc-pkcs11.so` here.  The debian package `opensc-pkcs11` installs it as `/usr/lib/x86_64-linux-gnu/opensc-pkcs11.so`.

Reply:
Updated README. Thanks.